src/chain/member_info.rs

```diff
@@ -54,15 +54,18 @@ pub struct MemberInfo {
     pub age_counter: AgeCounter,
     pub state: MemberState,
     pub p2p_node: P2pNode,
+    // Section version at the time this node joined.
+    pub section_version: u64,
 }
 
 impl MemberInfo {
     /// Create new `MemberInfo` in the `Joined` state.
-    pub fn new(age: u8, p2p_node: P2pNode) -> Self {
+    pub fn new(age: u8, p2p_node: P2pNode, section_version: u64) -> Self {
```

Comment:
this gives me a slight confusion that this `section_version` is our knowledge of that member's current section version.
However, I don't get a better name so far.  :(

Reply:
It's not the section_version we think the node has, rather it's the section_version at which the node joined which is something the whole section agrees on.

Reply:
Yeah, I know what it `want to do`.
it's just, within the chain::shared_state, there is a member field called the same name `section_version`. This causes some bit confustion to me at the beginning.

Reply:
OK. It should be something like `section_version_at_the_time_of_join`, but that seems too verbose :)

Reply:
yeah, I agree. I don't get a better name either. :(

Reply:
`joined_at_version: SectionVersion`, where `type SectionVersion = u64`